src/components/a-input/index.js

```diff
@@ -26,6 +29,9 @@ class AXAInput extends BaseComponentGlobal {
       'a-input--disabled': disabled,
     });
   }
+  didRenderCallback() {
```

Comment:
dont forget to cleanup events like in dropdown

Reply:
done